test/Shared/UserManagerTestBase.cs

```diff
@@ -39,7 +35,8 @@ protected virtual void SetupIdentityServices(IServiceCollection services, object
             services.AddIdentity<TUser, TRole>().AddDefaultTokenProviders();
             AddUserStore(services, context);
             AddRoleStore(services, context);
-            services.AddInstance<ILoggerFactory>(loggerFactory);
+            services.AddInstance<ILoggerFactory>(LoggerFactory);
```

Comment:
This is needed because its sticking a TestLoggingFactory in there will cause the normal ILoggerFactory to not get added(TryAdd will no-op), but we need AddLogging to bring in the `ILogger<>` service